docs/source/tutorials.rst

```diff
@@ -6,3 +6,160 @@ Tutorials
 .. contents::
     :local:
     :depth: 3
+
+To guide you through the learning curve of installation modules of birdhouse and set up an running birdhouse ecosystem, administer the server-side birdhouse components or even improve and develop your own specific functions, here are some general tutorials. This is a collection of tutorials and examples covering to show usage of WPS services but is also covering general data management with an focus on sustainable development.
+
+If you are a newcommer, you might have to go through a basic python tutorial first:
+.. todo:: Python tutorial
+
+Climate data management
+-----------------------
+**general Tutorial** about climate and related data handling.
+
+* esfg-python client
+
+
+WPS general usage
```

Comment:
There is also an outdated pywps/birdhouse workshop which could be linked:
https://birdhouse-workshop.readthedocs.io/en/latest/

Reply:
Great! 
That's going to be a section in the tutorials